### ISSUE TITLE
[bug] Incorrect indentation due to commented line(s) following a function call with a function argument.

### ISSUE THREAD

**Issue by trusktr** [beautify-web/js-beautify#713]
For example, if you have this code:

``` js
Meteor.call('foo',bar,function(err,result){
                                       Session.set('baz',result.lorem)
})
//blah blah
//blah blah
```

then js-beautify will turn it into this (give or take small differences depending on your indentation size setting, etc):

``` js
Meteor.call('foo', bar, function(err, result) {
        Session.set('baz', result.lorem)
    })
    //blah blah
    //blah blah
```

rather than what was expected:

``` js
Meteor.call('foo', bar, function(err, result) {
    Session.set('baz', result.lorem)
})
//blah blah
//blah blah
```

But if you start out with an extra new line between the function call and the comments, then you get the proper result, so this:

``` js
Meteor.call('foo', bar, function(err, result) {
                                    Session.set('baz', result.lorem)
})

//blah blah
//blah blah
```

turns into this:

``` js
Meteor.call('foo', bar, function(err, result) {
    Session.set('baz', result.lorem)
})

//blah blah
//blah blah
```

as expected.

**Comment by trusktr** [comment]
Turns out this bug happens also if there are zero new lines between the function call and the comment, so 

```js
Meteor.call('foo',bar[i].baz,{$inc:{'profile.lorem':1}}) //inc
```

turns into

```js
Meteor.call('foo', bar[i].baz, {
        $inc: {
            'profile.lorem': 1
        }
    }) //inc
```

instead of the (more) expected

```js
Meteor.call('foo', bar[i].baz, {
    $inc: {
        'profile.lorem': 1
    }
}) //inc
```

**Comment by bitwiseman** [comment]
The beautifier has always had a tough time with newline delimited statements.  :sad:

So, as you might imagine, this works: 
```js
Meteor.call('foo', bar[i].baz, {
    $inc: {
        'profile.lorem': 1
    }
});
//inc
```
This is a bug, but fixing it is not as simple as one would like it be.

**Comment by seoker** [comment]
Maybe same issue with https://github.com/Glavin001/atom-beautify/issues/494 ?

**Comment by bitwiseman** [comment]
This has the same cause as #964 and would also benefit from the fix proposed in #1069.
